tests/integration-tests/tests/proxy/test_proxy/test_proxy/pcluster.config.yaml

```diff
@@ -0,0 +1,26 @@
+Image:
+  Os: {{ os }}
+HeadNode:
```

Comment:
Let's add the policy `AmazonSSMManagedInstanceCore` to both the head node and compute node.
That policy is required for internal reporting and to allow SSMing into the instance for troubleshooting

Reply:
Done!